src/libstore/store-api.hh

```diff
@@ -552,7 +552,7 @@ void removeTempRoots();
 ref<Store> openStore(const std::string & uri = getEnv("NIX_REMOTE"));
 
 
-void copyPaths(ref<Store> from, ref<Store> to, const Paths & storePaths);
+void copyPaths(ref<Store> from, ref<Store> to, const Paths & storePaths, bool substitute = false);
```

Comment:
@k0001 FYI if this gets merged it will be easy to implement #937 in the new c++ build-remote